lib/services/role.go

```diff
@@ -167,26 +537,48 @@ type RoleV2 struct {
 	Spec RoleSpecV2 `json:"spec"`
 }
 
-// Equals returns true if roles are equal
+// Equals returns true if roles are equal.
```

Comment:
could be more meaningful comment - e.g. `Equals` test roles for equality, roles are considered equal if all rules and options match